example/c10_custom_cs_example.cpp

```diff
@@ -67,10 +51,10 @@ struct default_strategy<point_tag, point_tag, Point1, Point2, martian_tag, marti
 
 int main()
 {
-    typedef boost::geometry::model::point
+    using mars_point = boost::geometry::model::point
```

Comment:
Should we make this consistently in all examples of the library?

Reply:
In the end, yes. I usually update them now if they are at or close to touched code.